numba_cuda/numba/cuda/cudadrv/info.py

```diff
@@ -0,0 +1,19 @@
+# Maps CTK version -> (min supported cc, max supported cc) inclusive
+CTK_SUPPORTED = {
```

Comment:
Perhaps this can be renamed to `_CUDA_CC_MIN_MAX_SUPPORT` or a similarly descriptive (and private) name.
```suggestion
_CUDA_CC_MIN_MAX_SUPPORT = {
```